cli/types/index.d.ts

```diff
@@ -4,7 +4,7 @@
 //                 Mike Woudenberg <https://github.com/mikewoudenberg>
 //                 Robbert van Markus <https://github.com/rvanmarkus>
 //                 Nicholas Boll <https://github.com/nicholasboll>
-// TypeScript Version: 3.4
+// TypeScript Version: 3.9
```

Comment:
I hope we can go to 4.x soon... although last time I tried this (`dtslint` tool) had lots of issues.

Reply:
`4.7` for the CLI types does work, which is good. I just wonder if we would mark this as a breaking change when we do that?